### ISSUE TITLE
No description or website provided.

### ISSUE THREAD

**Issue by degendra** [bajratech/bajratech.github.io#5]
Please provide website link as https://bajratech.github.io and some other information if possible. It will be easy to navigate to website.

@bajratech/blog-team

**Comment by akrmhrjn** [comment]
Can you view the recent changes?? @degendra 
And where do you suggest to keep the link?

**Comment by degendra** [comment]
Yep, I can see new changes. Just below top navigation bar (i.e. code, issues, pull request etc.) we can provide link to the website. I believe it can be done by updating repository description. @akrmhrjn

**Comment by akrmhrjn** [comment]
Cool, that can be done. I was assuming you were telling me to put the link on the website. @degendra

**Comment by akrmhrjn** [comment]
I think that description must be updated by the admin of the repo. @nkarki 
@bajratech/blog-team
